packages/react/src/themes/teams/components/Dialog/dialogStyles.ts

```diff
@@ -11,11 +11,18 @@ export default {
     outline: 'none',
     padding: v.rootPadding,
     position: 'relative',
-    width: v.rootWidth,
+    width: v.rootWidthSmall,
     display: 'grid',
     gridTemplateColumns: '1fr auto',
     boxShadow: v.boxShadow,
     color: v.foregroundColor,
+
+    '@media (min-width: 768px)': {
+      width: v.rootWidthMedium,
+    },
+    '@media (min-width: 1280px)': {
+      width: v.rootWidth,
+    },
```

Comment:
@codepretty can you please provide proper breakpoints? 🙏

Reply:
Trying to nail down the breakpoint logic with Design... Almost have it figured out.